doc/man3/RSA_get0_key.pod

```diff
@@ -58,6 +58,10 @@ set with RSA_get0_factors() and RSA_set0_factors(), and the B<dmp1>,
 B<dmq1> and B<iqmp> parameters can be obtained and set with
 RSA_get0_crt_params() and RSA_set0_crt_params().
 
+For RSA_get0_key(), RSA_get0_factors(), and RSA_get0_crt_params(),  
+NULL value BIGNUM ** output parameters are permitted.  The functions 
+ignores NULL parameters but return values for other, non-NULL, parameters.
```

Comment:
repeat your edit/commit cycle.  Then do "git push -f github HEAD" to push the new tip of your branch to github.